src/tv2-common/helpers/graphics/internal/index.ts

```diff
@@ -31,6 +40,22 @@ import {
 import { GetInternalGraphicContentCaspar } from '../caspar'
 import { GetInternalGraphicContentVIZ } from '../viz'
 
+export interface InternalGraphic {
```

Comment:
I like this interface being used to minimise the number of function arguments, but then if all the functions including `getInternalGraphicContent`, `createAdlibTargetingOVL`, `createAdlib` and `createPiece` take it as their first parameter, shouldn't `InternalGraphic` be a class instead, and these functions be its methods?